server/mlabns/handlers/lookup.py

```diff
@@ -23,7 +23,7 @@ def _create_tool_url(fqdn, http_port):
     Returns:
         A constructed tool HTTP URL with protocol and http port.
     """
-    return 'http://%s:%s' % (fqdn, http_port)
+    return str('http://%s:%s' % (fqdn, http_port))
```

Comment:
This change looks redundant and like it does nothing?  Did you mean:

`return 'http://%s:%s' % (fqdn, str(http_port))`

or maybe

`return 'http://%s:%d' % (fqdn, http_port)`

?